src/set-core.js

```diff
@@ -2,10 +2,37 @@ var h = require("./helpers");
 var clause = require("./clause");
 var compare = require("./compare");
 var get = require("./get");
-var assign = require("can-util/js/assign/assign");
+var assign = require("can-assign");
 var each = require("can-util/js/each/each");
 var makeArray = require("can-util/js/make-array/make-array");
 var isEmptyObject = require("can-util/js/is-empty-object/is-empty-object");
+var getProps = require("can-util/js/get/get");
```

Comment:
I would alias this as `getPropValue` or `getProp` as it only returns a single property value.

Reply:
now getProp